packages/client/e2e/step-definitions/evaluation.spec.ts

```diff
@@ -24,7 +24,9 @@ const givenAnOpenfeatureClientIsRegisteredWithCacheDisabled = (
 defineFeature(feature, (test) => {
   beforeAll((done) => {
     client.addHandler(ProviderEvents.Ready, () => {
-      done();
+      setTimeout(() => {
+        done(); // TODO Fix by setting readiness correctly in provider
+      }, 2000);
```

Comment:
I don't quite understand this... can we avoid this timeout? Is this a TODO for this PR or to fix later?

Reply:
The spec says, that a provider without the status flag should be considered ready.
As the FlagdProvider certainly does not provide that right now, the code is directly called before the provider is initialized. This leads to the tests failing.
We could try to find a better way, but @beeme1mr and me discussed it and said, that we would just accept this now and fix the FlagD-Provider asap.

Reply:
https://github.com/open-feature/js-sdk/pull/429/files/59621b664d4edbc6a7692070799d9c82c0647fa6#r1204327488